Lanchonete/Lanchonete/ Features/Customization/Data/IngredientManager.swift

```diff
@@ -20,4 +20,31 @@ public class IngredientManager {
     public func containsIngredient(ingredient: Ingredient) -> Bool {
         return ingredients.contains(ingredient)
     }
+    
+    public func getQuantity(for ingredient: Ingredient) -> Int {
+        var quantity = 0
+        
+        for item in ingredients {
+            if item == ingredient {
+                quantity += 1
+            }
+        }
+        
+        return quantity
+    }
```

Comment:
Is this the same as `return ingredients.filter { $0 == ingredient }.count`?
It's funny how I always have to read these `for` loops like 10 times to ensure that there are no hidden nuances lol

Reply:
Yes it's the same, I think it'll be simpler if there's a filter instead of a for 😄  I'll add it now